docs/Microservices/MPS/securityMPS.md

```diff
@@ -37,7 +37,7 @@ To use secure protocols, MPS requires configured certificates and securely store
 For production deployment, purchase CA-signed certificates whose signatures can be independently verified.
 
 ### 5. Web User Credentials
-The Open AMT Cloud Toolkit is designed to operate behind an API gateway, such as Kong API Gateway. The API Gateway validates the Auth Tokens provided by an administrator who is requesting access to an API end point. Once verified, the API Gateway will forward the request to the appropriate microservice, MPS or RPS. To make evaluation easy, MPS has implemented an [Authorize](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/1.4.0#/Auth/post_api_v1_authorize) API end point that will issue a JWT when the proper web user credentials are provided. The Web User credentials are global credentials that are configured in the MPS configuration file and do not provide any fine-grain permissions.  Integration with other user authentication models and fine-grain endpoint permissions are supported through Kong [plug-ins](https://konghq.com/products/kong-gateway/kong-plugins/) and modification of the Kong API Gateway configuration file, respectively.
+The Open AMT Cloud Toolkit is designed to operate behind an API gateway, such as Kong API Gateway. The API Gateway validates the Auth Tokens provided by an administrator who is requesting access to an API end point. Once verified, the API Gateway will forward the request to the appropriate microservice, MPS or RPS. To make evaluation easy, MPS has implemented an [Authorize](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/{{ mpsAPI.version }}#/Auth/post_api_v1_authorize) API end point that will issue a JWT when the proper web user credentials are provided. The Web User credentials are global credentials that are configured in the MPS configuration file and do not provide any fine-grain permissions.  Integration with other user authentication models and fine-grain endpoint permissions are supported through Kong [plug-ins](https://konghq.com/products/kong-gateway/kong-plugins/) and modification of the Kong API Gateway configuration file, respectively.
```

Comment:
It's super minor.  There is endpoint and end point, both ways are correct, as far as the dictionary I consult is concerned. I'd choose one and go with it.